river/stats/lossy.py

```diff
@@ -0,0 +1,119 @@
+import math
+import typing
+from operator import itemgetter
+
+from river import stats
+
+
+class LossyCount(stats.base.Univariate):
+    """Lossy Counting with Forgetting factor[^1].
+
+    Keep track of the frequent itemsets in a data stream and apply a forgetting factor to
+    discard previous frequent items that do not often appear anymore.
```

Comment:
If I were I would expand this part a little bit to explain what are frequent itemsets, and how this differents to using `collections.Counter`.

Reply:
Done!